kubernetes/ansible/roles/secor-deploy/defaults/main.yaml

```diff
@@ -0,0 +1,27 @@
+secor_namespace: "secor"
+image:
+  prefix: {}
+  repository: "secor"
+  tag: 0.29
+  pullPolicy: IfNotPresent
+
+secor_jobs:
+  rawbackup:
+    replicas: 1
+    consumer_group: "{{env}}.telemetry.raw"
+    service_name: "raw_telemetry_backup"
+    base_path: "raw_secor_helm"
+    timestamp_key: "syncts"
+    fallback_timestamp_key: "@timestamp"
+    topic: "local.telemetry.raw"
```

Comment:
is this supposed to be `local` or `{{env}}`

Reply:
It is supposed to be {{ env }}. I will change it.